src/main/java/xyz/aadev/generitech/client/gui/machines/GuiPulverizer.java

```diff
@@ -41,7 +41,9 @@
 import net.minecraft.util.EnumFacing;
 import org.lwjgl.util.Point;
 import org.lwjgl.util.Rectangle;
+import scala.Int;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Remove this unused import 'scala.Int'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar.aadev.xyz/coding_rules#rule_key=squid%3AUselessImportCheck)